test/extended/olm/olmv1.go

```diff
@@ -20,66 +20,50 @@ import (
 
 const (
 	olmv1GroupName                       = "olm.operatorframework.io"
-	typeIncompatibelOperatorsUpgradeable = "InstalledOLMOperatorsUpgradeable"
+	typeIncompatibleOperatorsUpgradeable = "InstalledOLMOperatorsUpgradeable"
 )
 
-var _ = g.Describe("[sig-olmv1] OLMv1 CRDs", func() {
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM] OLMv1 CRDs", func() {
```

Comment:
@tmshort just confirming, we need `[OCPFeatureGate:NewOLM] ` so that we can continue run the test on default payload?

Reply:
This is also what the "5 e2es with 14 runs" check for promotion to default payload looks for.

Reply:
But yes, there is machinery that enables tests based on which feature gates are enabled in the cluster. That means, that whenever we add ourselves to the default payload, the test runner will know to run these tests against the default payload.